storage/src/create_bucket_turbo_replication.php

```diff
@@ -31,7 +31,7 @@
  * The bucket must be a dual-region bucket for this setting to take effect.
  *
  * @param string $bucketName The name of your Cloud Storage bucket.
- * @param string $location The Dual Region location where you want your bucket to reside. (Read more at https://cloud.google.com/storage/docs/locations#location-dr)
+ * @param string $location The Dual-Region location where you want your bucket to reside. (Read more at https://cloud.google.com/storage/docs/locations#location-dr)
```

Comment:
I'd like to see an example of the dual-region format here as well
```suggestion
 * @param string $location The Dual-Region location where you want your bucket to reside (e.g. "US-CENTRAL1+US-WEST1").
                                           Read more at https://cloud.google.com/storage/docs/locations#location-dr
```